samples/meeting-av-filters/js/README.md

```diff
@@ -0,0 +1,77 @@
+---
+page_type: sample
+description: "This sample illustrates usage of AV filters in Teams meeting."
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+- javascript
+extensions:
+  contentType: samples
+  createdDate: "10/04/2022 09:09:10 AM"
+urlFragment: officedev-microsoft-teams-samples-meeting-av-filters-javascript
+---
+
+# Meeting AV Filters
+
+This sample demos the use of AV filters for Teams meetings.
+
+![filter-app](Images/MeetingFilter.gif)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+2) Install node modules
+
+   Inside node js folder,  navigate to `samples/meeting-av-filters/nodejs/ClientApp` open your local terminal and run the below command to install node modules. You can do the same in Visual Studio code terminal by opening the project in Visual Studio code.
+
+    ```bash
+    npm install
+    ```
+3) We have the run the solution so follow below steps:
```

Comment:
Please review this sentence.

And, why step 3) is an individual step if what it does is merely navigating to the sample directory?

Reply:
Changed it